Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.3.md

```diff
@@ -14,34 +14,35 @@ As with all software, there is always a chance for issues and bugs, especially f
-   - Warning!! - This control is not finished and needs some work.
-7. Added ability to _`IContent`_ items to know if they have been created using **_Velaptor's_*** content pooling system.
-   - This ability gives the user the ability to know if a content item was created and added to the pooling system vs. created as a custom instantiation by the user.
-8. Added 2 new properties to the controls.
+2. Created scene management systems for the following testing applications:
+   - Text rendering
+   - Mouse functionality
+   - Keyboard functionality
+   - Graphics rendering
+3. Created a scene management system to help easily add new scenes to the testing application.
+4. Created new abstract control base class named _`Velaptor.UI.ControlBase`_ for the purpose of creating custom controls.
+5. Created new _`Velaptor.UI.Label`_ control.
+6. Created new _`Velaptor.UI.Button`_ control.
+7. Created new _`Velaptor.UI.Textbox`_ control.
+   > ⚠️ - This control is not finished and needs some work.
+8. Added ability to the _`IContent`_ items to know whether or not they have been created using **_Velaptor's_*** content pooling system.
+   - This enables the user to know if a content item was created and added to the pooling system or created as a custom instantiation by the user.
+9. Added 2 new properties to the controls.
    - _`MouseHoverColor`_ - This is the color that is applied on top of the control during rendering when the mouse hovers over the control.
    - _`MouseDown`_ - This is the color that is applied on top of the control during rendering when the left mouse button is in the down position over a control.
-9.  Added new boolean property named _`IsPooled`_ to the _`IContent`_ interface and all implementation members.
+10. Added new boolean property named _`IsPooled`_ to the _`IContent`_ interface and all implementation members.
     - This property value is true if it was created using **_Velaptor's__** content creation system.
-10. Created new exception type named _`PooledDisposalException`_.
-   - This exception is thrown if an _`IContent`_ item is being disposed when it was created with **_Velaptor's_** content creation system which uses content pooling for performance.
-   - NOTE: To know if an item was created in a pool, query the boolean value of the _`IContent.IsPooled`_ property.
-11. Added ability to _`IContent`_ items to throw an exception of type _`PooledDisposalException`_.
-12. Added ability for the content loaders below to take in a content name with or without and extension.
-    - NOTE: If a content name contains an extension, it is ignored.
+11. Created new exception type named _`PooledDisposalException`_.
+    - This exception is thrown if an _`IContent`_ item is being disposed when it was created with **_Velaptor's_** content creation system which uses content pooling for performance.
+      > **💡** To know if an item was created in a pool, query the boolean value of the _`IContent.IsPooled`_ property.
```

Comment:
You do not need to use the bold syntax for the light bulb.  This is because it is not text but an emoji so it doesn't actually bold.

I am sure that I have made this mistake as well and I will have to stop doing this as well.

Go through all of the release notes and remove any '*' that wrap emojis for note and warning blocks.